pywatts/modules/calendar_extraction.py

```diff
@@ -8,40 +9,63 @@
 from pywatts.core.exceptions.util_exception import UtilException
 from pywatts.core.exceptions.wrong_parameter_exception import WrongParameterException
 from pywatts.utils._workalendar_utils import _init_calendar
+from pywatts.utils._xarray_time_series_utils import _get_time_indeces
+
+
+class CalendarFeature(IntEnum):
```

Comment:
In my view, we should provide docstrings for this class to explain the provided features. For instance, we should explain the behavior of workday, weekend, and holiday since they are partially redundant, e.g. is_workingday returns false=0 on the weekend and on holidays.